config/src/main/java/com/quorum/tessera/config/constraints/ValidServerAddress.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 mark.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.quorum.tessera.config.constraints;
+
+import java.lang.annotation.Documented;
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import java.lang.annotation.Retention;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({FIELD, METHOD, PARAMETER, ANNOTATION_TYPE, TYPE_PARAMETER, TYPE_USE})
+@Retention(RUNTIME)
+@Constraint(validatedBy = ServerAddressValidator.class)
+@Documented
+public @interface ValidServerAddress {
+
+    boolean isBindingAddress() default false;
+    
+    String[] supportedSchemes() default {"unix","http","https"};
+    
+    String message() default "{ValidServerAddress.message}";
```

Comment:
Not sure this is used, but this message doesn't exist. Is it worth defining in the validation message list?